docker/docker-entrypoint.sh

```diff
@@ -21,6 +21,15 @@ if [ ! -z "${GEOLITE_LICENSE_KEY}" ]; then
   php bin/cli visit:download-db -n -q
 fi
 
+# Periodicaly run visit:locate every hour
+# https://shlink.io/documentation/long-running-tasks/#locate-visits
+# set env var "ENABLE_PERIODIC_VISIT_LOCATE=1" to enable
+if [ $ENABLE_PERIODIC_VISIT_LOCATE ]; then
+  echo "Starting periodic visite locate..."
+  echo "0 * * * * php bin/cli visit:locate -q" > /etc/crontabs/root
```

Comment:
Using relative paths in cron jobs is usually problematic. Instead of `bin/cli` I would use `/etc/shlink/bin/cli`.
```suggestion
  echo "0 * * * * php /etc/shlink/bin/cli visit:locate -q" > /etc/crontabs/root
```